### PR TITLE
Fixes #1348 remove tags

### LINKED ISSUE #1348: Cleanup tags tables
image

Seems to be a lot of useless stuff there (historically this was potentially required but not anymore I believe)

/cc @Yuri05 ?

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/Db/Dump/PKSimDB_dump.txt [hunk 1]
```diff
@@ -90174,6 +89670,508 @@ INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSum
 INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSumFormulas','PARAM_Q_lng','Lung','NotInContainer');
 INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSumFormulas','PARAM_Q_lng','Pericentral','NotInContainer');
 INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSumFormulas','PARAM_Q_lng','Blood flow rate','Has');
+CREATE TABLE IF NOT EXISTS "tab_tags" (
```
Comment (Yuri05) [inline src/Db/Dump/PKSimDB_dump.txt]: "If not exists"? interesting...

File: src/Db/Dump/PKSimDB_dump.txt [hunk 2]
```diff
@@ -90174,6 +89670,508 @@ INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSum
 INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSumFormulas','PARAM_Q_lng','Lung','NotInContainer');
 INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSumFormulas','PARAM_Q_lng','Pericentral','NotInContainer');
 INSERT INTO tab_calculation_method_rate_descriptor_conditions VALUES('DynamicSumFormulas','PARAM_Q_lng','Blood flow rate','Has');
+CREATE TABLE IF NOT EXISTS "tab_tags" (
+	"tag"	text NOT NULL,
+	PRIMARY KEY("tag")
+);
```
Comment (Yuri05) [inline src/Db/Dump/PKSimDB_dump.txt]: I would keep the description column, just no need for NOT NULL condition there

**Comment by msevestre** [comment]
@Yuri05 DEscription was never used so far (always set to tag), I'd suggest to approve this PR and we can add a description column back that will be empty

**Comment by Yuri05** [comment]
agree

**Pull request merged**